enterprise/frontend/test/metabase-enterprise/audit/auditing.cy.spec.js

```diff
@@ -13,108 +11,81 @@ import { SAMPLE_DATASET } from "__support__/cypress_sample_dataset";
 const { PRODUCTS } = SAMPLE_DATASET;
 
 const year = new Date().getFullYear();
-
-export function generateQuestions(users) {
-  users.forEach(user => {
-    signIn(user);
-
-    cy.request("POST", `/api/card`, {
-      name: `${user} test q`,
-      dataset_query: {
-        type: "native",
-        native: {
-          query: "SELECT * FROM products WHERE {{ID}}",
-          "template-tags": {
-            ID: {
-              id: "6b8b10ef-0104-1047-1e1b-2492d5954322",
-              name: "ID",
-              display_name: "ID",
-              type: "dimension",
-              dimension: ["field-id", PRODUCTS.ID],
-              "widget-type": "category",
-              default: null,
-            },
+const QUESTION = "question";
+const DASHBOARD = "dashboard";
+
+export function generateQuestions(user) {
```

Comment:
Just realized these functions were being exported. As soon as CI finishes running, I'll fix this.